hack/verify-kiali-server-permissions.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+!/bin/bash
```

Comment:
Is this right? I've never see this without the comment # prefix

Reply:
seems a typo